src/output.c

```diff
@@ -163,7 +163,8 @@ EXPORT void write_metainfo(FILE *f, struct metafile *m, unsigned char *hash_stri
 				(unsigned long)strlen(m->comment),
 				m->comment);
 	/* I made this! */
-	fprintf(f, "10:created by13:mktorrent " VERSION);
+    char const *const created_by = "mktorrent " VERSION;
```

Comment:
It's a nitpick, but I don't see why it needs to be a const pointer?

Reply:
@uno20001

It doesn't, but it doesn't need to _not_ be a `const` pointer, right? I tend to make everything as `const` as possible by default, especially in cases like these, where this is really meant to be just an immutable string.

Reply:
I understand that, however, it struck me as odd since this is the only occurence of such construct in the codebase as far as I know. I just wanted to know if you had any reason in mind when writing the code.